packages/simcore-sdk/src/simcore_sdk/node_ports_v2/r_clone.py

```diff
@@ -0,0 +1,212 @@
+import asyncio
+import logging
+import re
+import urllib.parse
+from contextlib import asynccontextmanager
+from pathlib import Path
+from typing import AsyncGenerator, Optional
+
+from aiofiles import tempfile
+from aiohttp import ClientSession, ClientTimeout, web
+from cache import AsyncLRU
+from settings_library.r_clone import RCloneSettings
+from settings_library.utils_r_clone import get_r_clone_config
+
+from ..node_ports_common.filemanager import ETag
+
+logger = logging.getLogger(__name__)
+
+
+class _CommandFailedException(Exception):
+    pass
+
+
+class RCloneError(Exception):
+    pass
+
+
+@asynccontextmanager
+async def _config_file(config: str) -> AsyncGenerator[str, None]:
+    async with tempfile.NamedTemporaryFile("w") as f:
+        await f.write(config)
+        await f.flush()
+        yield f.name
+
+
+async def _async_command(command: str, *, cwd: Optional[str] = None) -> str:
```

Comment:
and command should be a list or a tuple.

Reply:
To answer your questions:

1) With `create_subprocess_shell` (which is the one I require here), we have something in the `dynamic-sidecar`, but has a very different interface.

2) The underlying `asyncio.create_subprocess_shell` expects a string. But I've moved it to List, since in the future this will probably change.